src/main/kotlin/com/team4099/robot2022/subsystems/Wheel.kt

```diff
@@ -205,16 +203,15 @@ class Wheel(
           acceleration
         }
     directionSetPoint = directionSensor.position + directionDifference
-    //    driveSpark.set(speedSetPoint / Constants.Drivetrain.DRIVE_SETPOINT_MAX)
+    //    driveFalcon.set(speedSetPoint / Constants.Drivetrain.DRIVE_SETPOINT_MAX)
 
-    drivePID.setReference(
+    driveFalcon.set(
+        ControlMode.Velocity,
         driveSensor.velocityToRawUnits(speedSetPoint),
-        ControlType.kVelocity,
-        0,
+        DemandType.ArbitraryFeedForward,
```

Comment:
can we confirm that the CTRE stuff wants the arbitrary feedforward to be in volts? I would have checked the docs directly but they seem to be down :/